zephyr/CMakeLists.txt

```diff
@@ -497,30 +512,6 @@ zephyr_library_sources_ifdef(CONFIG_SAMPLE_KEYPHRASE
 	${SOF_SAMPLES_PATH}/audio/detect_test.c
 )
 
-if(CONFIG_IPC_MAJOR_3)
```

Comment:
Conversion of a specific module to loadable should be in separate commit than the loadable module infrastructure